### ISSUE TITLE
Ammonite gives a File Not Found error on MacOS Mojave

### ISSUE THREAD

**Issue by werezak** [lihaoyi/Ammonite#913]
When I run the command from the documentation:

`sudo sh -c '(echo "#!/usr/bin/env sh" && curl -L https://github.com/lihaoyi/Ammonite/releases/download/1.6.0/2.12-1.6.0) > /usr/local/bin/amm && chmod +x /usr/local/bin/amm' && amm`

on MacOS 10.14.2 with Scala 2.12.18 and Java 1.8.0_181-b13, I get:

```
Loading...
Exception in thread "main" java.nio.file.NoSuchFileException: /Users/user/.ammonite/session
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at os.write$.write(ReadWriteOps.scala:61)
	at os.write$over$.apply(ReadWriteOps.scala:137)
	at ammonite.runtime.Storage$Folder.getSessionId(Storage.scala:132)
	at ammonite.Main.run(Main.scala:185)
	at ammonite.MainRunner.$anonfun$runRepl$1(Main.scala:374)
	at ammonite.MainRunner.watchLoop(Main.scala:355)
	at ammonite.MainRunner.runRepl(Main.scala:374)
	at ammonite.Main$.main0(Main.scala:291)
	at ammonite.Main$.main(Main.scala:255)
	at ammonite.Main.main(Main.scala)
```

**Comment by werezak** [comment]
Oh, I see that this error is the subject of pull request #912.

**Comment by kiendang** [comment]
To work around this, just create the `/Users/user/.ammonite` directory.

**Comment by werezak** [comment]
Thank you!  (I should have thought of that.) Now it's working.

I tried compiling it with:

`sbt clean compile package`

but perhaps there is more to it than that, because it would not compile:

```
[info] Loading project definition from /Users/user/Downloads/Ammonite-master/project
[info] Updating {file:/Users/user/Downloads/Ammonite-master/project/}ammonite-master-build...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Set current project to ammonite-master (in build file:/Users/user/Downloads/Ammonite-master/)
[info] Executing in batch mode. For better performance use sbt's shell
[success] Total time: 0 s, completed 5-Jan-2019 7:07:38 PM
Generating Scalatex Sources...
[info] Updating {file:/Users/user/Downloads/Ammonite-master/}ammonite-master...
[info] Updating {file:/Users/user/Downloads/Ammonite-master/}readme...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Resolving jline#jline;2.14.4 ...
[info] Done updating.
java.util.NoSuchElementException: key not found: CONSTANTS_FILE
	at scala.collection.MapLike$class.default(MapLike.scala:228)
	at scala.collection.AbstractMap.default(Map.scala:58)
	at scala.collection.MapLike$class.apply(MapLike.scala:141)
	at scala.collection.AbstractMap.apply(Map.scala:58)
	at $073515eda795fd0f3049$$anonfun$readme$6.apply(build.sbt:17)
	at $073515eda795fd0f3049$$anonfun$readme$6.apply(build.sbt:17)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[error] (readme/compile:unmanagedSources) java.util.NoSuchElementException: key not found: CONSTANTS_FILE
[error] Total time: 0 s, completed 5-Jan-2019 7:07:38 PM
```

**Comment by kiendang** [comment]
I think sbt build is not supported anymore. You have to use mill instead.

**Comment by TJC** [comment]
Issue occurs for me too, on Ubuntu 18.04 LTS Bionic, with Azul JDK 11.0.1

**Comment by lihaoyi** [comment]
Fixed in master; should be out in 1.6.1 which I just kicked off the build for
